service-info/README.md

```diff
@@ -0,0 +1,55 @@
+# GA4GH Service Info Types
+
+This file details how to add values to the list of Service Info type values for the GA4GH API.
```

Comment:
It would be good to add details of: Who should raise the PR to add particular values? (Presumably the working group for the protocol in question…) When should it be raised?

Reply:
Yes, agree the Working Group in question can raise this. I feel the natural timing for this would be in the run-up to the Product Approval Process form being submitted, so can expand that documentation accordingly.

Reply:
Service Info added to the Product Approval form and the Product Approval Process Guide, which can have links added to the files once the PR is merged.